common/persistence/nosql/nosqlplugin/cassandra/gocql/tracing.go

```diff
@@ -0,0 +1,141 @@
+package gocql
+
+import (
+	"context"
+	"fmt"
+	"net"
+	reflect "reflect"
+	"strings"
+	"time"
+
+	"github.com/gocql/gocql"
+	"go.opentelemetry.io/otel/attribute"
+	semconv "go.opentelemetry.io/otel/semconv/v1.12.0"
```

Comment:
```suggestion
	"go.opentelemetry.io/otel/semconv/v1.12.0"
```

Reply:
goimports requires this one because that's not actually a module version number. https://github.com/golang/go/issues/30051